### PR TITLE
Review-2

### PR DESCRIPTION
**Pull request by pmaxy**
Things done:
Created feature-2 branch
Updated files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,11 +34,11 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 Hello Microverse <a name="about-project"></a>
+# 📖 Review Past project <a name="about-project"></a>
 
 > Introduction to the first project.
 
-**Portiflio Setup and mobile first** is a introductory project to test skills for basic concepts of programming
+**Review Past project** is a project of checking and reviewing practice in microverse
```
Comment (pmaxy) [inline README.md:42]: Changed the heading of the readme to much the project name.
💯

File: index.html [hunk 2]
```diff
@@ -72,7 +72,7 @@ <h3>Reciving Retro PCs</h3>
         <h2>02</h2>
         <h3>Top 10 Laptops of 2022</h3>
         <p>
-            Our best packs for various needs and budgets.
+          Our best packs for various needs and budgets.
         </p>
```
Comment (pmaxy) [inline index.html:76]: Removed spacing on the to paragraph below to maintaining the microverse standard of 2

**Review by pmaxy — COMMENTED** [review]
hello @pmaxy

Great job so far! 👏
You've done it well

# **To highlight** 🎯

- [ ] - No linter errors.✔️
- [ ] - Nice design. ✔️
- [ ] - Descriptive PR. ✔️

unnamed

# **Approved** 👏 
 
# Optional suggestions

- N/A

**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

<hr>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).<hr>

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hi @pmaxy,

Everything is looking great! Please, merge this branch :shipit:
Congratulations! 🎉

## Highlights

 - Good use of inline comments. ✔️
 - Professional comment. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**